packages/components/src/components/PipelineRun/PipelineRun.js

```diff
@@ -226,6 +226,7 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
               definition={definition}
               logContainer={logContainer}
               reason={reason}
+              showIO
```

Comment:
Doesn't this need to come from a parameter in this component too ? Given the is the one consumed at the top level